tests/fsm/test_regex.py

```diff
@@ -18,8 +18,8 @@
     reduced_vocabulary,
     walk_fsm,
 )
-from outlines.integrations.utils import adapt_tokenizer
 from outlines.models.transformers import TransformerTokenizer
+from outlines.models.vllm import adapt_tokenizer
```

Comment:
`test_reduced_vocabulary_with_rare_tokens` should just use `TransformerTokenizer` rather than calling `adapt_tokenizer`